torch/csrc/jit/script/compiler.cpp

```diff
@@ -2880,7 +2880,13 @@ struct to_ir {
       // if it's a list, emit a regular index selection op
       auto* idx = emitExpr(subscript_exprs[0]);
       return emitBuiltinCall(
-          loc, *graph, aten::select, c10::nullopt, {gatherable, idx}, {}, true);
+          loc,
```

Comment:
Merged them together, I also tried to merge the tuple indexing, but it might resides on multiple sugared values so I decided to still leave it alone.